test/cases/koji.sh

```diff
@@ -122,9 +135,12 @@ sudo cp \
     /etc/pki/ca-trust/source/anchors/osbuild-composer-tests-ca-crt.pem
 sudo update-ca-trust
 
-greenprint "Restarting composer to pick up new config"
+start_jwt_auth_server
+
+greenprint "Restarting composer, stopping a local worker and starting a remote worker"
 sudo systemctl restart osbuild-composer
-sudo systemctl restart osbuild-worker\@1
+sudo systemctl stop osbuild-worker@1
```

Comment:
Local worker doesn't work at all with JWT, right? So maybe let's call this in `provision.sh` when `jwt` is requested?

Reply:
Yes, that sounds reasonable. The provision.sh does weird thing TBH, it starts the remote worker `.socket` unit, but it does not mask the local worker. So in reality, they both may be running  as part of some test cases.